parseURL.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import "github.com/xmidt-org/bascule/basculehttp"
+
+type parseURLOption struct {
+	URLPathPrefix string
+}
+
+func parseURLFunc(o parseURLOption) basculehttp.ParseURL {
+	return basculehttp.CreateRemovePrefixURLFunc(o.URLPathPrefix, basculehttp.DefaultParseURLFunc)
```

Comment:
Does this work if the path prefix is an empty string? (If not, that's probably something we should fix in the `CreateRemovePrefixURLFunc()`)

Reply:
I forgot to add the `/` as the default value for the prefix. I'll add that. 
As a general comment: This new approach of reading config with arrange allows us to define defaults pretty easily. For few fields like in this case, adding the values inline works and for bigger structs, we could create helper functions like `defaultxyzConfig()`. This is much easier than having to check if values are unset after the unmarshaling happens 🎉